Proiect-VS/assets/css/login.less

```diff
@@ -0,0 +1,116 @@
+body {
+    margin: auto;
+}
+
+.skin_body {
+    font-family: Arial;
+}
+
+.banner {
+    position: relative;
+    opacity: 0.6;
+} 
+
+.login {
+    position: absolute;
+    top: 80px;
+    left: 300px;
+    width: 300px;
+    height: 250px;
+    border: none;
+    border-radius: 3px;
+}
+
+.skin_login {
+     background-color: white;
+     box-shadow: 0px 2px 2px #333435;   
+}
+
+.login_header {    
+    position: absolute; 
+    width:100%;
+    border-radius: 3px;
+    padding-top: 25px;
+    text-align: center;
+    vertical-align: middle;
+    line-height: 20px;
+    height: 40px;
+}
+
+.skin_login_header {
+    background-color: #419fe2; 
+    color: white;
+    font-size: 20px;
+}
+
+.login_input {
```

Comment:
General feedback: in general '-' convention is used in CSS( eg login-input etc )